build_docs.py

```diff
@@ -98,8 +159,9 @@ def build_one(version, isdev, quick, sphinxbuild, build_root, www_root,
     maketarget = "autobuild-" + ("dev" if isdev else "stable") + ("-html" if quick else "")
     logging.info("Running make %s", maketarget)
     logname = os.path.basename(checkout) + ".log"
-    shell_out("cd Doc; make SPHINXBUILD=%s %s >> %s 2>&1" %
-              (sphinxbuild, maketarget, os.path.join(log_directory, logname)))
+    shell_out("cd Doc; make SPHINXBUILD=%s SPHINXOPTS='%s' %s >> %s 2>&1" %
+              (sphinxbuild, sphinxopts, maketarget,
+               os.path.join(log_directory, logname)))
```

Comment:
That's right, it will even allow running make directly (without a shell) once your previous comment fixed, which is nice. I tried to keep my changes minimal so I didn't changed this. Will do in another PR, later.

(As I don't have access to the logs or the server, I prefer keeping my chances atomic, so if something break we all know exactly what caused it).